documentation/dev/plugins.rst

```diff
@@ -150,12 +150,28 @@ This will find `css/styles.css` if you add that folder and file to your Blueprin
 .. note:: This styling will only apply to the pages defined in your plugin (to pages based on your own base template). To apply a styling to all other pages which are served by FlexMeasures, consider using the config setting :ref:`extra-css-config`. 
 
 
-Adding config options
+Adding config settings
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-You might want to override some FlexMEasures configuration settings from within your plugin. Some examples for possible settings are named on this page, e.g. the custom style (see above) or custom logo (see below). There is a `record_once` function on Blueprints which can help with this. Example:
+FlexMeasures can be made to check for any custom config settings your plugin is using.
```

Comment:
I think this would be better:

> FlexMeasures can automatically check for you if any custom config settings, which your plugin is using, are present. This can be very useful in maintaining installations of FlexMeasures with plugins.